### ISSUE TITLE
Clearer and more consistent interface for adding Death Shroud to players

### ISSUE THREAD

**Issue by Smutchings** [ThePandemoniumInstitute/botc-release#110]
**Is your feature request related to a problem? Please describe.**
Sometimes the death shroud doesn't show when you hover over the Player's circle/camera feed. 

**Describe the solution you'd like**
Add a dedicated interface for adding the Death shroud to a player.
Activate this interface with a keyboard command or by clicking on the player you wish to "kill" and being able to select an option from the dropdown list that normally comes up.
Tapping the keyboard command would allow you to click on a player and add the death shroud.
Clicking on a player's space in the Grimoire would allow you to add the death shroud.

**Describe alternatives you've considered**
Current alternative is to hover over the top of the player's camera feed and click the shroud icon. But this doesn't always appear and isn't an obvious UI.

**Additional context**
N/A

**Comment by bra1n** [comment]
Agreed, adding shrouds to players with a camera feed isn't currently very easy to do and could be improved. I'll try to allow adding a shroud by clicking on the cam feed as well, but that'll take a bit of time.

**Comment by gareth** [comment]
I think the core problem in this issue disappeared when user avatars were added in September 2024 - there's no longer a fundamentally different UX for players with cameras enabled vs those without.

The keyboard shortcut suggestion wasn't implemented, but since there is now a clear and consistent interface for adding a death shroud I'm going to close this issue as resolved.
